site/_includes/partials/chrome-100-card-section.njk

```diff
@@ -0,0 +1,2584 @@
+{% block content %}
+<card-section class="card-section">
+  <div class="navigation-wrapper">
+    <img src="/images/meta/chrome_2008.png" alt="Chrome logo" class="chrome-logo">
```

Comment:
Please move all of the `chrome_YYYY.png` images to the CDN as well.

Reply:
Done

Reply:
We have moved all the images except for this as we need this static to manipulate the code through JS.

Reply:
Please remove those as well. You can e.g. a dics of urls instead:

```
{ 
'chrome_100: 'some-hardcoded-url-from-cdn.png',
'chrome_200: 'some-another-hardcoded-url-from-cdn.png',
}
```

Checking in multiple images per the same logo version is highly suboptimal.

Reply:
Updated the code.